src/Microsoft.Android.Build.BaseTasks/Files.cs

```diff
@@ -28,6 +38,33 @@ public static class Files
 		public static readonly Encoding UTF8withoutBOM = new UTF8Encoding (encoderShouldEmitUTF8Identifier: false);
 		readonly static byte[] Utf8Preamble = Encoding.UTF8.GetPreamble ();
 
+		/// <summary>
+		/// Checks for the environment variable DOTNET_ANDROID_FILE_WRITE_RETRY_ATTEMPTS to
+		/// see if a custom value for the number of times to retry writing a file has been 
+		/// set.
+		/// </summary>
+		/// <returns>The value of DOTNET_ANDROID_FILE_WRITE_RETRY_ATTEMPTS or the default of DEFAULT_FILE_WRITE_RETRY_ATTEMPTS</returns>
+		public static int GetFileWriteRetryAttempts ()
```

Comment:
I made these public so we can use them from XA.

Reply:
Do we have an other environment variables like this? I wonder if they should go in a class like `KnownEnvironment.cs`.

Reply:
We don't afaik.